docs/gettingstarted/installation.rst

```diff
@@ -151,12 +151,18 @@ Using Amazon Web Services
 .. note:: If you are not familiar with EC2 you may want to `read more about the terminology and concepts <https://fedoraproject.org/wiki/User:Gholms/EC2_Primer>`_ used in this document.
           You can also refer to `the full documentation for interacting with EC2 from Amazon Web Services <http://docs.amazonwebservices.com/AWSEC2/latest/GettingStartedGuide/>`_.
 
-#. Choose an AMI for your EC2 Instance
-
-   * Visit https://fedoraproject.org/en/get-fedora#clouds
-   * In the EC2 **Fedora 20** section choose your local region and **64 Bit** and click the "Launch it!" button.
-
-#. Configure the AMI
+#. Choose a region for your EC2 instance
```

Comment:
I had a little confusion first time I read this, it's not clear that clicking the links actually chooses for you as opposed to just being a list of regions Amazon provides. Also, it takes you away from this page so maybe it should open new page?